examples/gocean/eg5/extract/Makefile

```diff
@@ -95,20 +96,18 @@ $(NAME): $(INF_LIB) $(EXTRACT_DIR)/lib_kernel_data_netcdf.a $(KERNELS) alg.o psy
 	$(INF_LIB)                                \
 	$$(nf-config --flibs)
 
-$(DRIVER_INIT):	$(KERNELS) $(DRIVER_INIT).o \
-				$(EXTRACT_DIR)/lib_kernel_data_netcdf.a
+$(DRIVER_INIT):	$(KERNELS) $(DRIVER_INIT).o
```

Comment:
I think this is missing a dependence on `$(READ_DIR)/read_kernel_data_mod.o`. I get (when doing `cd examples; make allclean; make compile`):

    gfortran -g -O0 $(nf-config --fflags) -I../../../../external/dl_esm_inf/finite_difference/src -I../../../../lib/extract/netcdf/dl_esm_inf -I../../../../lib/extract/netcdf -c driver-main-init.f90
    driver-main-init.f90:2:6:

    2 |   use read_kernel_data_mod, only : ReadKernelDataType
      |      1
    Fatal Error: Cannot open module file ‘read_kernel_data_mod.mod’ for reading at (